helm-chart/mezod/README.md

```diff
@@ -0,0 +1,61 @@
+# Helm chart for Mezod
+
+The chart contains configuration for testnet on Sepolia so far.
+Eventually, it will be updated to support mainnet by default.
```

Comment:
Can we list what needs to be updated for mainnet? Just to have it in once place once the time comes.

Reply:
By writing that, I meant that in the future, we will replace default values in values.yaml with ones for mainnet—for example, the network and Docker image.
It might be that we'll change some other things as well, but it's hard to say that now.